src/ModelObjects/component.rs

```diff
@@ -257,6 +257,18 @@ impl Component {
         self.input_edges = Some(i_edges);
     }
 
+    /// Redoes the components Edge IDs by giving them new unique IDs based on their index.
+    pub fn remake_edge_ids(&mut self) {
+        // Give all edges a name
+        self.edges[0].id = "a".to_string();
```

Comment:
```suggestion
```

Reply:
fixed, forgot to do it through suggestion feature